pkg/quarterdeck/orgs.go

```diff
@@ -159,21 +160,21 @@ func (s *Server) ProjectCreate(c *gin.Context) {
 	// Bind the Project request to the project data structure
 	if err = c.BindJSON(&project); err != nil {
 		sentry.Warn(c).Err(err).Msg("could not parse project create request")
-		c.JSON(http.StatusBadRequest, api.ErrorResponse(api.ErrUnparsable))
+		c.JSON(http.StatusBadRequest, api.ErrorResponse(responses.ErrTryProjectAgain))
 		return
 	}
 
 	// Validate the request from the API side.
 	if err = project.Validate(); err != nil {
 		c.Error(err)
-		c.JSON(http.StatusBadRequest, api.ErrorResponse(err))
+		c.JSON(http.StatusBadRequest, api.ErrorResponse(responses.ErrTryProjectAgain))
```

Comment:
Since this is an error related to validation, would it be helpful to have a message informing users a project can't be created with the details provided, in a way similar to `ErrTryLoginAgain`, or would that give too much away?

Reply:
Hmm, possibly; it might depend on what triggers the validation failures in practice, probably Beacon should be doing form validation before sending requests to QD or Tenant, and those validation errors can help guide the user to fill in fields that they've left empty.